spec/spec_helper.rb

```diff
@@ -3,6 +3,12 @@
 require 'webmock/rspec'
 require 'slimmer/rspec'
 require 'capybara/rails'
+require 'capybara/poltergeist'
+
+WebMock.disable_net_connect!(allow_localhost: true)
```

Comment:
This was copied from government-frontend https://github.com/alphagov/government-frontend/blob/master/test/support/webmock.rb